ui/inspectors/system-section.reel/docker-settings.reel/docker-settings.js

```diff
@@ -9,16 +9,28 @@ var Component = require("montage/ui/component").Component,
  * @extends Component
  */
 exports.DockerSettings = Component.specialize(/** @lends DockerSettings# */ {
+
+    apiForwardingAddress: {
+        value: null
+    },
+
     enterDocument: {
         value: function(isFirstTime) {
             var self = this;
+                this.apiForwardingAddress = [];
             if(isFirstTime) {
                 this.isLoading = true;
                 this.application.dockerSettingsService.getDockerHostQueryData().then(function(dockerHosts) {
                     self.dockerHosts = dockerHosts;
                 })
                 this.application.dockerSettingsService.getDockerConfigData().then(function (dockerConfig) {
-                    self.object = dockerConfig
+                    self.object = dockerConfig;
+                    var api_forwarding_list = dockerConfig.api_forwarding.split(",")
```

Comment:
Check if `dockerConfig.api_forwarding` is defined and is it is a string before calling `.split`